etc/templates/compute.glue21

```diff
@@ -221,6 +221,13 @@ GLUE2EndpointTrustedCA: ${trusted_ca}
         share_max_vm = quotas['instances']
     else:
         share_max_vm = False
+
+    fields =  {'sla', 'network_info'}
+    for field in fields:
+      if field not in share or share[field] is None:
+          share[field] = 'UNKNOWN'
+
```

Comment:
It should be possible to remove some of those empty lines, not sure they are all useful/required.

Reply:
Removed them